src/en/news/blog/2022/scaletesting-with-pawsey/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Quincy @ Scale: Testing with the Pawsey Supercomputing Centre"
+date: "2022-07-11"
+author: "Paul Cuzner"
+tags:
+  - scalability
+  - quincy
+  - cephadm
+---
+
+During the development of new features, one of the most difficult considerations for Ceph developers is whether a new feature needs to be validated at scale. Our labs allow us to test new features in clusters of up to X,000 OSDs, and in most circumstances this works well!
+
+However, our developers are a cautious bunch and like to test beyond our typical 'test ceiling', if the opportunity arises.
+
+Late last year, such an opportunity presented itself. The **[Pawsey Supercomputing Centre](https://pawsey.org.au/)** based in Perth, Australia reached out with a potential test window. They identified several weeks in their rollout plan where their hardware was ‘racked and stacked’ but would remain unused.
+
+As we were approaching the end of the development cycle for the Quincy release, this was an opportunity too good to miss!
+
+---
+
+## Hardware
+
+Pawsey provided access to over **200** servers, all connected via 100Gb networking. The server specifications were optimized for a large Object storage use case, and enabled us to create the following Ceph cluster;
+
+- 180 OSD Hosts
+- 4,320 OSDs
+- 69 PB (_raw capacity_)
+- 100Gb Networking (_multiple VLANs_)
+- Separate mon/mgr nodes (_'service'_ nodes)
+
+## Test Objectives
+
+Although the orchestrator/cephadm control plane has been in place since the Ceph Octopus release, the challenge of deploying and managing a cluster of this size was defined as our main goal.
```

Comment:
main goal of this testing, cephadm or both?

Reply:
rephrased - thanks.